opentelemetry-distro/src/opentelemetry/distro/__init__.py

```diff
@@ -89,7 +82,9 @@ def _init_tracing(
             EXPORTER_OTLP,
             EXPORTER_OTLP_SPAN,
         ]:
-            exporter_args["service_name"] = service_name
+            exporter_args["service_name"] = provider.resource.attributes.get(
+                SERVICE_NAME
+            )
```

Comment:
Why is this needed anymore? I would have expected some test fail here. I believe there aren't enough tests for distro.

Reply:
The dummy exporter class's __init__() accepted a service_name. hence the tests weren't failing. 
I will remove this and also the service_name from the class exporter __init__